### PR TITLE
Ignore unneeded files from Sublime package

### LINKED ISSUE #379: No need to package `tests` folder in Sublime package
### Sublime Text Version
Version 3.2.1, Build 3207

### Rust Enhanced Version
v2.19.0

### Operating system
All

### Expected behavior
There is no need to package `tests` folder in Sublime package.

**Benefit**: Reduce package's size.

### Actual behavior
```bash
% unzip -l Rust\ Enhanced.sublime-package |rg '\s+tests'
    21826  2019-05-16 19:27   tests/test_cargo_build.py
      540  2019-05-16 19:27   tests/README.md
     8696  2019-05-16 19:27   tests/test_interrupt.py
     2376  2019-05-16 19:27   tests/test_click_handler.py
     2468  2019-05-16 19:27   tests/test_phantom_style.py
     2062  2019-05-16 19:27   tests/test_cargo_settings.py
     3323  2019-05-16 19:27   tests/test_target_detect.py
...
```

### PR DESCRIPTION
**Pull request by lzutao**
Closes #379

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Used to remove files from deployment using `git archive`
+# and from GitHub release compressed files.
+
+# git files
+.gitattributes export-ignore
+.gitignore export-ignore
+
+# Ignored folders
+.github export-ignore
+changelog export-ignore
```
Comment (ehuss) [inline .gitattributes]: `changelog` shouldn't be ignored, it is used by package control.

File: .gitattributes [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+docs export-ignore
+tests export-ignore
+
+# CI files
+.travis.yml export-ignore
+appveyor.yml export-ignore
+bors.toml export-ignore
+
+# general text files
+LICENSE.txt export-ignore
```
Comment (ehuss) [inline .gitattributes]: I think the license should be kept.

**Comment by lzutao** [comment]
@ehuss  Could you set Travis to auto cancel old PR builds?

**Comment by ehuss** [comment]
> @ehuss Could you set Travis to auto cancel old PR builds?

I turned it on, but I don't think it will help much, since AFAIK travis does not cancel running builds.

**Comment by lzutao** [comment]
Oh, GitHub release's zip files seem to ignore intended files.
https://github.com/lzutao/rust-enhanced/releases/tag/v2.19.1

**Pull request merged**